airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -133,6 +133,7 @@ class DbtCloudHook(HttpHook):
     Interact with dbt Cloud using the V2 API.
 
     :param dbt_cloud_conn_id: The ID of the :ref:`dbt Cloud connection <howto/connection:dbt-cloud>`.
+    :param tenant: The subdomain / tenancy of a dbt Cloud account.
```

Comment:
```suggestion
```
Can you remove the `tenant` param from the hook and sensor?

Reply:
Done! Thanks for catching